server/src/views/shell.js

```diff
@@ -17,7 +17,9 @@ export class Shell extends React.Component {
       <script src={ this.props.staticLink("vendor/introjs/intro.js") } key="introjs-js" />,
       <script src={ this.props.staticLink("js/server-bundle.js") } key="server-bundle-js" />,
     ];
-    js = js.concat(gaActivation(this.props.gaId, this.props.deviceId, false));
+    if (this.props.gaId) {
```

Comment:
I had intended (but probably forgot) to make gaActivation put in a stub `ga()` function when gaId wasn't provided.  Then this could be unconditionally included.